server/src/main/java/org/elasticsearch/script/ScriptedMetricAggContexts.java

```diff
@@ -31,6 +33,25 @@
 import java.util.Map;
 
 public class ScriptedMetricAggContexts {
+    private static final DeprecationLogger DEPRECATION_LOGGER =
+        new DeprecationLogger(Loggers.getLogger(ScriptedMetricAggContexts.class));
+
+    // Public for access from tests
+    public static final String AGG_PARAM_DEPRECATION_WARNING =
+        "params._agg/_aggs for scripted metric aggregations are deprecated, use state/states (not in params) instead. " +
+        "Set system property es.aggregations.enable_scripted_metric_agg_param = false to disable this deprecated behavior.";
```

Comment:
We typically give the actual command line, something like:

```
-Des.aggregations.enable_scripted_metric_agg_param = false to jvm.options to disable...
```

Reply:
Updated based on the format of the joda time ScriptDocValues deprecation message that's also currently active.